src/steamtricks

```diff
@@ -429,36 +451,67 @@ steamtricks_handle_option()
   return 0
 }
 
+
 while steamtricks_handle_option $1 ; do
   shift
 done
 
+if test "$STEAMTRICKS_WATCH" == 1 ; then
+  #redirect stdout as watch daemon
+  log="$CONFIG_DIR/steamtricks.log"
+  exec 1<&-
+  exec 1<>$log
+fi
+
 # pid file to detect already running
 if [ -f "$PID_FILE" ] ; then
-  echo "steamtricks is already running"
-  exit 1
+  #running but want to stop
+  if test "$STEAMTRICKS_DAEMON_STOP" == 1 ; then
+    echo "Stopping steamtricks..."
+    pkill -F "$PID_FILE"
+    exit 0
+  fi
+  
+  #ignore if running watch thread
+  if test "$STEAMTRICKS_WATCH" == 0 ; then
+    echo "steamtricks is already running"
+    exit 1
+  fi
+  
+else
+  if test "$STEAMTRICKS_DAEMON_STOP" == 1 ; then
+   echo "steamtricks is not currently running"
+   exit 1
+  fi
 fi
-trap "rm -f -- '$PID_FILE'" EXIT
-echo $$ > "$PID_FILE"
 
 # initial boot
 steamtricks_config
+
+#check if daemon requested
 if test "$STEAMTRICKS_DAEMON" == 1 ; then
   log="$CONFIG_DIR/steamtricks.log"
   if [ -f "$log" ] ; then
     mv "$log" "$log.1"
   fi
-  ($0 --watch &> "$log" &)
-  echo "steamtricks launched"
+  
+  #launch watch instance in background
+  $0 --watch &
+  echo $! > "$PID_FILE"
```

Comment:
Should this be changed to `$$` like `--pid` on watch above? Seems like reviewing the meaning `$$` revers to bash pid, vs `$!` pid of last job run in background.